.github/workflows/nodejs.yml

```diff
@@ -13,6 +13,4 @@ jobs:
     uses: stylelint/.github/.github/workflows/lint.yml@main
 
   test:
-    uses: stylelint/.github/.github/workflows/test.yml@main
-    with:
-      os: '["ubuntu-latest", "windows-latest", "macos-latest"]'
+    uses: stylelint/.github/.github/workflows/test.yml@remove-nodejs-12 # TODO: Change to `@main`.
```

Comment:
[note] https://github.com/stylelint/.github/pull/15

Also, I believe we don't need Windows and macOS because this package is very simple.